docs/source/guides/build.rst

```diff
@@ -385,4 +386,4 @@ If you want to explore the maximum of Windows-related works around
 Iroha please take a look at these pull requests:
 `1 <https://github.com/hyperledger-archives/iroha/pull/1988>`_,
 `2 <https://github.com/hyperledger-archives/iroha/pull/2022>`_,
-`3 <https://github.com/hyperledger/iroha/pull/55>`_.
\ No newline at end of file
+`3 <https://github.com/hyperledger/iroha/pull/55>`_.
```

Comment:
Unable to recognize what is changed here.

Reply:
https://patch-diff.githubusercontent.com/raw/hyperledger/iroha/pull/87.patch
There was "No newline at end of file"